### PR TITLE
merge/aave3-main

### PR DESCRIPTION
**Pull request by shmuel-web**
fixed conflicts

### CODE REVIEW

File: backgroundJobs/Addresses.js [hunk 1]
```diff
@@ -39,14 +39,19 @@ const oneInchOracleAddress = {
...
+	"ETH": "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696",
+	"BSC": "0xcb6E2F66DF0493b4dD23ac7727D1677D1208b697",
+	"AVAX": "0xcb6E2F66DF0493b4dD23ac7727D1677D1208b697",
+	"MATIC": "0xe539B93620570A92EF90eF7E60c79d9fAee91186",
+	"NEAR": "0xcb6E2F66DF0493b4dD23ac7727D1677D1208b697",
+	"FTM": "0xab35D115974aC0A3C4Bd16A70DF77003c9f4C011",
+	"ARBITRUM": "0x0c05e6968AED7Ca120464a78731144e75052CeB1",
+	"CRO": "0x5e954f5972EC6BFc7dECd75779F10d848230345F",
+	"MOONBEAM": "0x6477204E12A7236b9619385ea453F370aD897bb2",
+    "OPTIMISM": "0xcA11bde05977b3631167028862bE2a173976CA11"
```
Comment (yaronvel) [inline backgroundJobs/Addresses.js:52]: fix indentation

File: backgroundJobs/Addresses.js [hunk 2]
```diff
@@ -126,10 +133,67 @@ const tectonicAddress = { "CRO" : { "comptroller" : "0xb3831584acb95ED9cCb0C11f6
...
 
+///////////// AAVE V3 SPECIFICS //////////////////////////
+
+const aaveV3Configuration =
+{
+	"AVAX": {
+		"poolAddressesProviderRegistry": "0x770ef9f4fe897e59daCc474EF11238303F9552b6",
+		"deployBlock": 11970175,
+		"blockStepInInit": 1000,
+        "firstEventBlock": 12027175,
```
Comment (yaronvel) [inline backgroundJobs/Addresses.js:144]: fix indentation, also below

**Review by yaronvel — CHANGES_REQUESTED** [review]

File: backgroundJobs/Addresses.js (hunk 2 above)
Comment (alfulinku) [reply backgroundJobs/Addresses.js:144]: Indentation is good locally, github probably display it wrong

File: backgroundJobs/Addresses.js (hunk 1 above)
Comment (alfulinku) [reply backgroundJobs/Addresses.js:52]: Indentation is good locally, github probably display it wrong

**Review by yaronvel — APPROVED** [review]

**Pull request merged**